app.js

```diff
@@ -24,4 +24,7 @@ app.use("/", indexRouter);
 app.use("/users", usersRouter);
 app.use("/courses", coursesRouter);
 
+const port = 7000;
```

Comment:
Please, could you remove these parts with listening ? It could causes some problems and it's not necessary